util/libcrypto.num

```diff
@@ -5425,44 +5425,44 @@ ASN1_item_d2i_ex                        5552	3_0_0	EXIST::FUNCTION:
-X509_PUBKEY_set0_public_key             ?	3_1_0	EXIST::FUNCTION:
-OSSL_STACK_OF_X509_free                 ?	3_1_0	EXIST::FUNCTION:
-EVP_MD_CTX_dup                          ?	3_1_0	EXIST::FUNCTION:
-EVP_CIPHER_CTX_dup                      ?	3_1_0	EXIST::FUNCTION:
-BN_signed_bin2bn                        ?	3_1_0	EXIST::FUNCTION:
-BN_signed_bn2bin                        ?	3_1_0	EXIST::FUNCTION:
-BN_signed_lebin2bn                      ?	3_1_0	EXIST::FUNCTION:
-BN_signed_bn2lebin                      ?	3_1_0	EXIST::FUNCTION:
-BN_signed_native2bn                     ?	3_1_0	EXIST::FUNCTION:
-BN_signed_bn2native                     ?	3_1_0	EXIST::FUNCTION:
-ASYNC_set_mem_functions                 ?	3_1_0	EXIST::FUNCTION:
-ASYNC_get_mem_functions                 ?	3_1_0	EXIST::FUNCTION:
-BIO_ADDR_dup                            ?	3_1_0	EXIST::FUNCTION:SOCK
-OSSL_CMP_CTX_get0_validatedSrvCert      ?	3_1_0	EXIST::FUNCTION:CMP
-OSSL_CRMF_CERTTEMPLATE_get0_publicKey   ?	3_1_0	EXIST::FUNCTION:CRMF
-CMS_final_digest                        ?	3_1_0	EXIST::FUNCTION:CMS
-CMS_EnvelopedData_it                    ?	3_1_0	EXIST::FUNCTION:CMS
-CMS_EnvelopedData_decrypt               ?	3_1_0	EXIST::FUNCTION:CMS
-CMS_SignedData_free                     ?	3_1_0	EXIST::FUNCTION:CMS
-CMS_SignedData_new                      ?	3_1_0	EXIST::FUNCTION:CMS
-CMS_SignedData_verify                   ?	3_1_0	EXIST::FUNCTION:CMS
+X509_PUBKEY_set0_public_key             ?	3_2_0	EXIST::FUNCTION:
+OSSL_STACK_OF_X509_free                 ?	3_2_0	EXIST::FUNCTION:
+EVP_MD_CTX_dup                          ?	3_2_0	EXIST::FUNCTION:
+EVP_CIPHER_CTX_dup                      ?	3_2_0	EXIST::FUNCTION:
+BN_signed_bin2bn                        ?	3_2_0	EXIST::FUNCTION:
+BN_signed_bn2bin                        ?	3_2_0	EXIST::FUNCTION:
+BN_signed_lebin2bn                      ?	3_2_0	EXIST::FUNCTION:
+BN_signed_bn2lebin                      ?	3_2_0	EXIST::FUNCTION:
+BN_signed_native2bn                     ?	3_2_0	EXIST::FUNCTION:
+BN_signed_bn2native                     ?	3_2_0	EXIST::FUNCTION:
+ASYNC_set_mem_functions                 ?	3_2_0	EXIST::FUNCTION:
+ASYNC_get_mem_functions                 ?	3_2_0	EXIST::FUNCTION:
+BIO_ADDR_dup                            ?	3_2_0	EXIST::FUNCTION:SOCK
+OSSL_CMP_CTX_get0_validatedSrvCert      ?	3_2_0	EXIST::FUNCTION:CMP
+OSSL_CRMF_CERTTEMPLATE_get0_publicKey   ?	3_2_0	EXIST::FUNCTION:CRMF
+CMS_final_digest                        ?	3_2_0	EXIST::FUNCTION:CMS
+CMS_EnvelopedData_it                    ?	3_2_0	EXIST::FUNCTION:CMS
+CMS_EnvelopedData_decrypt               ?	3_2_0	EXIST::FUNCTION:CMS
+CMS_SignedData_free                     ?	3_2_0	EXIST::FUNCTION:CMS
+CMS_SignedData_new                      ?	3_2_0	EXIST::FUNCTION:CMS
+CMS_SignedData_verify                   ?	3_2_0	EXIST::FUNCTION:CMS
 OPENSSL_strcasecmp                      ?	3_0_3	EXIST::FUNCTION:
 OPENSSL_strncasecmp                     ?	3_0_3	EXIST::FUNCTION:
```

Comment:
Shouldn't these be assigned the same numbers as they were in 3.0.3?

Reply:
No, this is what they are in the `openssl-3.0` branch:

https://github.com/openssl/openssl/blob/6b9d582d49353750f36c14dd20f5f962e7d52d3b/util/libcrypto.num#L5428-L5429

Reply:
Argh, you're right.  I zeroed in on *version* numbers [sigh]

Reply:
I made a separate PR for this: #19357